posts/auditing-using-javers.md

```diff
@@ -0,0 +1,238 @@
+---
+authors:
+- Amine Ouali Alami
+- Pooja Krishnan
+tags:
+- HIPAA
+- PHI
+- Audit Trail
+- JaVers
+- Spring Data
+- Spring Boot
+- JHipster
+
+date: 2020-01-10T20:16:10.000Z
+title: "Audit your data with JaVers"
+image: https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2020/01/audit-trail.jpg
+---
+
+As an IT consultant, the first requirements that comes to mind when you are working in the Healthcare Industry is Health Insurance Portability and Accountability Act (HIPAA). The technology plays a significant role to ensure data remains secure and HIPAA-compliant. According to cybercrime experts, stolen Protected Health Information (PHI) is 10 times more valuable than credit card information.
```

Comment:
Is there a linkable source for the claim in the last sentence?

Reply:
I found many articles that's talk about this: 

https://www.reuters.com/article/us-cybersecurity-hospitals-idUSKCN0HJ21I20140924
https://www.justjohncrowley.com/why-phi-is-10-times-more-valuable-than-a-ssn-on-the-black-market/
https://www.infoworld.com/article/2983634/why-hackers-want-your-health-care-data-breaches-most-of-all.html
https://www.fortherecordmag.com/archives/0316p18.shtml

I couldn't find an official government report that confirms this, I'll keep looking.

Reply:
Yes, I finally found a report from the FBI that is mentioning that (http://www.illuminweb.com/wp-content/uploads/ill-mo-uploads/103/2418/health-systems-cyber-intrusions.pdf) Page 2 first paragraph.
Do you want me to add a link to that report?

Reply:
Yes please!